ScoutSuite/providers/aws/aws_facade.py

```diff
@@ -0,0 +1,34 @@
+import boto3
+from botocore.session import Session
+from collections import Counter
+
+# TODO: Handle authentication better. I don't even know how it currently works. I think connect_service is called somewhere.
+class AwsFacade(object):
```

Comment:
Can we keep the provider names uppercase for consistency (`AWS`, `GCP`, `Azure`)? Details, details...

Reply:
Of course, I never noticed AWSProvider was uppercase, my bad.